CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Add `retrieveMe()` Convenience Function that allow users to retrieve without specifying an ID.
 - Removed `enable()` and `disable()` methods in the apiKey class. Please use this functionality through EasyPost website
 - Removed `options.useCookie` from the parameter since it's deprecated in v4.0. Please use `options.useProxy` instead.
+- Fixes `customs_item.value` to ve a `number` instead of a `string`
```

Comment:
```suggestion
- Fixes `customs_item.value` to be a `number` instead of a `string`
```

Reply:
Oof, thanks. I'll fix this when we release it and I rework the CHANGELOG so I don't dismiss the review.